how-we-work/iteration.qmd

```diff
@@ -104,7 +110,7 @@ the iteration as well as the distribution of priority labels of the
 tasks.
 
 Then, we agree on and schedule the end date debrief and retrospective
-meeting, which should be approximately 2-3 hours and preferably before
+meeting, which should be approximately 1-2 hours and preferably before
 lunch, as well as the next iterations planning meeting. The next
 planning meeting should either be on the same date after lunch, to
 ensure a proper break before moving to a new iteration, or the next day
```

Comment:
```suggestion
lunch, as well as the next iterations planning meeting. We have found that the
planning meeting is best held the day after to ensure that we don't get too tired towards the end. Alternatively it can be the same day as the debrief, but there should be a substantial break between the two meetings.
```